### PR TITLE
Refactor activemq_xml check

### PR DESCRIPTION
**Pull request by remh**
Also add safeguards in case of crazy amount of queues/topics/subscribers

### CODE REVIEW

File: checks.d/activemq_xml.py [hunk 1]
```diff
@@ -20,17 +36,25 @@ def check(self, instance):
         url = instance.get("url")
         username = instance.get("username")
         password = instance.get("password")
+        custom_tags = instance.get('tags', [])
+        max_queues = int(instance.get("mqx_queues", MAX_ELEMENTS))
```
Comment (LeoCavaille) [inline checks.d/activemq_xml.py]: typo `max_queues`

File: checks.d/activemq_xml.py [hunk 2]
```diff
@@ -11,6 +11,22 @@
...
+    "size": "size"
+}
+
+SUBSCRIBER_TAGS = [
+            "connectionId",
+            "subscriptionName",
+            "destinationName",
+            "selector",
+            "active",
+        ]
```
Comment (LeoCavaille) [inline checks.d/activemq_xml.py]: nitpicky whitespacing, needs alignment :wink:

File: conf.d/activemq_xml.yaml.example [hunk 3]
```diff
@@ -1,8 +1,21 @@
...
+       # - queue2
+
+
+       #detailed_topics: # Optional. If you have more than 300 topics you need to list the ones you want to track
+       # - topic1
+       # - topic2
+
+       #detailed_subscribers: # Optional. If you have more than 300 subscribers you need to list the ones you want to track
+       # - subscriber1
+       # - subscriber2
```
Comment (LeoCavaille) [inline conf.d/activemq_xml.yaml.example]: Could you make this YAML file follow Mike's refactor rules?

**Comment by LeoCavaille** [comment]
Good for me, except the few remarks above, thanks @remh

**Comment by remh** [comment]
Thanks for the review @LeoCavaille . Your comments are fixed.

**Pull request merged**